CanDrive.js

```diff
@@ -0,0 +1,7 @@
+var FirstName = "Aissatou";
+var LastName = "Diallo";
+var Age = 23;
```

Comment:
it's good in JavaScript to use camelCase to name your variables.

firstName
lastName
age
etc...